lib/onc_certification_g10_test_kit/unrestricted_resource_type_access_group.rb

```diff
@@ -371,5 +371,27 @@ def resource_group
         USCoreTestKit::USCoreV311::ProcedureGroup
       end
     end
+
+    if Feature.us_core_v4?
+      test from: :g10_resource_access_test do
+        title 'Access to ServiceRequest resources granted'
+        description %(
+          This test ensures that access to the ServiceRequest is granted or
+          denied based on the selection by the tester prior to the execution of
+          the test. If the tester indicated that access will be granted to this
+          resource, this test verifies that a search by patient in this resource
+          does not result in an access denied result. If the tester indicated that
+          access will be denied for this resource, this verifies that search by
+          patient in the resource results in an access denied result.
```

Comment:
We need to update all the descriptions in this set of tests because there is no "selection by the tester prior to the execution of the test" for the unrestricted type.  Seems like an oversight in v2.0.